### ISSUE TITLE
Genkai Tokki Moero Crystal H 極限凸起 萌情水晶 H 한계돌파 모에로크리스탈 H - 0100D7E0110B2000

### ISSUE THREAD

**Issue by yuwenhua653** [Ryujinx/Ryujinx-Games-List#3683]
## Game Name
Genkai Tokki Moero Crystal H 
極限凸起 萌情水晶 H 
한계돌파 모에로크리스탈 H
#### Current on `master`  : *1.0.7008*
Like the US version, the game has no graphics and audio output at all

#### Screenshots :
![屏幕截图 2021-08-23 083009](https://user-images.githubusercontent.com/30205483/130375389-f809b006-4259-4e7b-9e7d-ab17569f4b1a.png)
#### Log file :
[Ryujinx_1.0.7008_2021-08-23_08-17-47.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/7028411/Ryujinx_1.0.7008_2021-08-23_08-17-47.log)

**Comment by yuwenhua653** [comment]
update

**Comment by SilverBeamx** [comment]
The game is playable, altough graphics are not rendering in the 3D sections of the game (ie. Dungeons). Only the 2D Sprite elements rendered on top of it are present. Moreover, performance is also poor (15-20 fps), which further plummets to around 2-3 fps depending on where the player is looking in the (blank) 3D environment. Here's a screenshot of the issue:
![moeroBug](https://user-images.githubusercontent.com/44263986/171898840-1a5e94d7-dc3f-4fc2-af27-322674c6556c.png)

Until these issues are fixed i would tag this game as "ingame" rather than "playable".

**Comment by SilverBeamx** [comment]
Here it is. Just pressing continue should load the player in that exact spot:
[MoeroSave.zip](https://github.com/Ryujinx/Ryujinx-Games-List/files/8834183/MoeroSave.zip)

**Comment by yuwenhua653** [comment]
> The game is playable, altough graphics are not rendering in the 3D sections of the game (ie. Dungeons). Only the 2D Sprite elements rendered on top of it are present. Moreover, performance is also poor (15-20 fps), which further plummets to around 2-3 fps depending on where the player is looking in the (blank) 3D environment. Here's a screenshot of the issue: ![moeroBug](https://user-images.githubusercontent.com/44263986/171898840-1a5e94d7-dc3f-4fc2-af27-322674c6556c.png)
> 
> Until these issues are fixed i would tag this game as "ingame" rather than "playable".

For the English version, you should file an issue here https://github.com/Ryujinx/Ryujinx-Games-List/issues/1769

**Comment by SilverBeamx** [comment]
You are right, i didn't notice that the game had different codes depending on the Localisation. Although, i think that the issue should be present on all versions, regardless of language. Could anyone with the Asian version check this?

**Comment by yuwenhua653** [comment]
> You are right, i didn't notice that the game had different codes depending on the Localisation. Although, i think that the issue should be present on all versions, regardless of language. Could anyone with the Asian version check this?

I use the archive you provided, same problem, I have written above

**Comment by gdkchan** [comment]
Changed the status back to playable since the 3D maze issue should be fixed as of emulator version 1.1.142. Please update the e main issue and let me know if you have any more issues. Thank you.
